src/AboutMe/aboutMe.css

```diff
@@ -0,0 +1,115 @@
+.main {
+  width: 90%;
+  padding: 20px 10px;
+  margin: 0px auto;
+  margin-top: 4rem;
+}
+.greet-main{
+  width: 90%;
+  padding: 20px 10px;
+  margin: 0px auto;
+  margin-top: 2rem;
+}
+
+.gretting-main {
```

Comment:
what is this class referring to? dont see an element in aboutMe.js with className gretting-main